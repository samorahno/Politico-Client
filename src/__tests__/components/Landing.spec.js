import React from 'react';
import { shallow } from 'enzyme';
import Landing from '../../components/landing/landing';

describe('Landing Page Component', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Landing />);
        expect(component).toMatchSnapshot();
    });
});

