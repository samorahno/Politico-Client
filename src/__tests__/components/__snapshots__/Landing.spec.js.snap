// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Page Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
